cli/cli.go

```diff
@@ -172,6 +173,14 @@ func SetupCLI(args []string) error {
 				}
 				return runOptions.handleCLIOptions(ctx)
 			},
+			Flags: []cli.Flag{
+				&cli.BoolFlag{
+					Name:        "reboot-exit-code",
+					Destination: &runOptions.rebootExitCode,
+					Usage: "Set exit code to 4 if a manual reboot " +
```

Comment:
Suggestion:
`Return exit code 4 If a manual reboot is required after the Artifact installation.`

Reply:
Notice the Uppercase `Artifact` especially.

Reply:
Done.